### PR TITLE
Add initial Feast of Hemlock Vale player cards

### PR DESCRIPTION
**Pull request by zzorba**
No investigators yet, as we don't have deckbuilding rules

### CODE REVIEW

File: packs.json [hunk 1]
```diff
@@ -427,6 +427,22 @@
...
+      "date_release": "2023-01-30",
+      "name": "The Feast of Hemlock Vale Investigator Expansion",
+      "position": 1,
+      "size": 300
+    },
+    {
+        "code": "fhvc",
+        "cycle_code": "fhv",
+        "date_release": "2022-02-28",
+        "name": "The Feast of Hemlock Vales Campaign Expansion",
```
Comment (drawntotheflame) [inline packs.json:442]:
```suggestion
        "name": "The Feast of Hemlock Vale Campaign Expansion",
```

File: pack/fhv/fhvp.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    "deck_limit": 2,
+    "faction_code": "seeker",
+    "illustrator": "Pixoloid Studios",
+    "name": "Chemistry Set",
+    "pack_code": "fhvp",
+    "position": 40,
+    "quantity": 2,
+    "skill_intellect": 1,
+    "slot": "Accessory",
+    "text": "[action] Exhaust Chemistry Set: <b>Investigate.</b> If you...\n- ...fail by exactly 2, discard Chemistry Set.\n- ...succeed by exactly 0, gain 2 resources.\n- ...succeed by exactly 2, draw 1 card.\n- ...succeed by exactly, 4 discover 1 additional clue at your location.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:14]:
```suggestion
    "text": "[action] Exhaust Chemistry Set: <b>Investigate.</b> If you...\n- ...fail by exactly 2, discard Chemistry Set.\n- ...succeed by exactly 0, gain 2 resources.\n- ...succeed by exactly 2, draw 1 card.\n- ...succeed by exactly 4, discover 1 additional clue at your location.",
```

File: pack/fhv/fhvp.json [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    "deck_limit": 2,
+    "faction_code": "seeker",
+    "illustrator": "Jason Caffoe",
+    "name": "Microscope",
+    "pack_code": "fhvp",
+    "position": 42,
+    "quantity": 2,
+    "skill_intellect": 1,
+    "slot": "Hand",
+    "text": "[reaction] After an enemy at your location is successfully evaded or defeated, exhaust Microscope: Place 1 resource on Microscope, as evidence.\n[action][action]: <b>Investigate.</b> You get +1 [intellect] for this investigation for each evidence on Microscope (max +3 [intellect]). If you succeed, you may spend up to 2 evidence to discover that many additional clues at your location.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:53]:
```suggestion
    "text": "[reaction] After an enemy at your location is successfully evaded or defeated, exhaust Microscope: Place 1 resource on Microscope, as evidence.\n[action] [action]: <b>Investigate.</b> You get +1 [intellect] for this investigation for each evidence on Microscope (max +3 [intellect]). If you succeed, you may spend up to 2 evidence to discover that many additional clues at your location.",
```

File: pack/fhv/fhvp.json [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+    "health": 1,
+    "illustrator": "JB Casacop",
+    "name": "Ravenous Myconid",
+    "pack_code": "fhvp",
+    "position": 44,
+    "quantity": 1,
+    "sanity": 1,
+    "skill_agility": 1,
+    "subname": "Unidentified",
+    "text": "Limit 1 per deck.\n[action]: Search your bonded cards for Uncanny Growth and add it to your hand.\n[action] If Ravenous Myconid has 3 or more growth on it: Move each growth to your resource pool, as resources. Record in your Campaign Log that \"you have classited a new species.\"",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:72]:
```suggestion
    "text": "Limit 1 per deck.\n[action]: Search your bonded cards for Uncanny Growth and add it to your hand.\n[action] If Ravenous Myconid has 3 or more growth on it: Move each growth to your resource pool, as resources. Record in your Campaign Log that \"you have classified a new species.\"",
```

File: pack/fhv/fhvp.json [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+    "traits": "Creature. Monster Flora. Science.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:73]:
```suggestion
    "traits": "Creature. Monster. Flora. Science.",
```

File: pack/fhv/fhvp.json [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+    "deck_limit": 2,
+    "faction_code": "seeker",
+    "flavor": "Every cent of that research grant counts.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Well-Funded",
+    "pack_code": "fhvp",
+    "position": 51,
+    "quantity": 2,
+    "skill_wild": 1,
+    "text": "While you control a [[Science]] or [[Tool]] asset, Well-Funded gains [wild] (while you control 3 or more [[Science]] and/or [[Tool]] assets, Well-Funded gains [wild][wild], instead).",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:103]:
```suggestion
    "text": "While you control a [[Science]] or [[Tool]] asset, Well-Funded gains [wild] (while you control 3 or more [[Science]] and/or [[Tool]] assets, Well-Funded gains [wild] [wild], instead).",
```

File: pack/fhv/fhvp.json [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+    "faction_code": "survivor",
+    "illustrator": "Pixoloid Studios",
+    "name": "Pitchfork",
+    "pack_code": "fhvp",
+    "position": 110,
+    "quantity": 2,
+    "skill_agility": 1,
+    "skill_combat": 1,
+    "slot": "Hand x2",
+    "text": "[action]: <b>Fight.</b> You get +1 [combat] and deal +2 damage for this attack. If this attack is successful, lose control of Pitchfork and attach it to your location. Your location gains:\n\"[action]: Take control of Pitchfork. Any investigator at Pitchfork's location may trigger this ability\"",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:138]:
```suggestion
    "text": "[action]: <b>Fight.</b> You get +1 [combat] and deal +2 damage for this attack. If this attack is successful, lose control of Pitchfork and attach it to your location. Your location gains:\n\"[action]: Take control of Pitchfork. Any investigator at Pitchfork's location may trigger this ability.\"",
```

File: pack/fhv/fhvp.json [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+    "traits": "Item. Tool. Weapon. Melee.",
+    "type_code": "asset",
+    "xp": 0
+  },
+  {
+    "code": "10111",
+    "cost": 1,
+    "deck_limit": 2,
+    "faction_code": "survivor",
+    "illustrator": "Rob Laskey",
```
Comment (drawntotheflame) [inline pack/fhv/fhvp.json:148]: Should this be 'Robert' so that we don't end up with both Rob Laskey and Robert Laskey as illustrator entries on the db?

**Review by drawntotheflame — COMMENTED** [review]
Great job @zzorba . Please tag Kam when these are fixed and this can go live.

File: pack/fhv/fhvp.json (hunk 8 above)
Comment (zzorba) [reply pack/fhv/fhvp.json:148]: Yeah, it changed on the card so I wasn't sure?  I think there have been instances of people changing their names in the past (to abbreviations or whatnot), and we've gone with it?

Comment (drawntotheflame) [reply pack/fhv/fhvp.json:148]: Then yeah, I think we just leave it...

**Comment by zzorba** [comment]
@Kamalisk this can go live now

**Pull request merged**